etc/c/zlib.d

```diff
@@ -80,7 +80,7 @@ alias void  function (void* opaque, void* address) free_func;
 
 struct z_stream
 {
-    ubyte*   next_in;  /* next input byte */
+    const(ubyte)*   next_in;  /* next input byte */
```

Comment:
On Thu, Jul 23, 2015 at 10:59:22AM -0700, Martin Nowak wrote:
> Is this in accordance with the zlib C header?

Yes, I linked to the documentation in the PR description,
you can double check it there.